Readability.js

```diff
@@ -1722,6 +1722,22 @@ Readability.prototype = {
   isProbablyReaderable: function(helperIsVisible) {
     var nodes = this._getAllNodesWithTag(this._doc, ["p", "pre"]);
 
+    // Get <div> nodes which have <br> node(s) and append them into the `nodes` variable.
+    // Some articles' DOM structures might look like
+    // <div>
+    //   Sentences<br>
+    //   <br>
+    //   Sentences<br>
+    // </div>
+    var brNodes = this._getAllNodesWithTag(this._doc, ["br"]);
```

Comment:
In practice isProbablyReaderable is only ever invoked on a "real" DOM, not on a JSDOMParser one. So I would go with the `div > br` option and update the test runner to only test `isProbablyReaderable` against a `jsdom` DOM (so `if` it out in the `JSDOMParser` case). Would that make sense?